gensrc/proto/cloud.proto

```diff
@@ -187,6 +190,25 @@ message ObjectStoreInfoPB {
     optional bool sse_enabled = 15;
 }
 
+message StorageVaultPB {
+    optional string id = 1;
+    optional string name = 2;
+    optional HdfsInfo hdfs_infos = 3; // HdfsResource
+}
+
+message HdfsInfo {
+    optional string fs_name = 1;
```

Comment:
We should have a field `path` in `HdfsInfo`, rather then in `hdfs_confs`.
`path` is the place to save data, we should use the `path` value to create dir or list files. This is one of the arguments of HDFS client methods.
`hdfs_confs` is the configs we create a HDFS client.